src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -20,21 +30,68 @@ test.describe("Buzz Page", () => {
 
   test.afterEach(async ({ page }) => {
     buzzPage = new BuzzPage(page);
-    await buzzPage.deleteTheNewestPost()
+    await buzzPage.deleteTheNewestPost(false)
   });
 
   test("Post should be shared", async ({ page }) => {
+    const randomTitle = generateRandomString()
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePost(filePath);
+    await buzzPage.sharePost(filePath, randomTitle);
     await page.reload();
-    await expect(buzzPage.photoImg.first()).toBeAttached();
+    await expect(buzzPage.bodyTextPost.first()).toHaveText(randomTitle);
   });
 
   test("Post should be edited", async ({ page }) => {
+    const randomTitle = generateRandomString()
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePost(filePath);
+    await buzzPage.sharePost(filePath, randomTitle);
     await page.reload();
     await buzzPage.editTheNewestPost(expectedPostTextAfterEdition)
-    await expect(page.getByText(expectedPostTextAfterEdition)).toBeVisible();
+    await expect(buzzPage.bodyTextPost.first()).toHaveText(expectedPostTextAfterEdition);
+  });
+
+  test("Video should be shared", async ({ page }) => {
+    await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
+    await buzzPage.shareVideo(videoTitle, videoUrl);
+    await expect(buzzPage.bodyTextPost.first()).toHaveText(videoTitle);
+  });
+});
+
+test.describe("Most liked and most commented post", () => {
+  let loginPage: LoginPage;
+  let buzzPage: BuzzPage;
+
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    buzzPage = new BuzzPage(page);
+    await loginPage.initialize();
+    await loginPage.navigateToMainPage();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+    await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
+    await buzzPage.shareVideo(videoTitle, videoUrl);
+    await buzzPage.shareVideo(mostLiked, videoUrl2);
+    await buzzPage.shareVideo(mostCommented, videoUrl3);
+    
+  });
+
+  test.afterEach(async ({ page }) => {
+    buzzPage = new BuzzPage(page);
+    await buzzPage.deleteTheNewestPost(true)
+    await buzzPage.deleteTheNewestPost(true)
+    await buzzPage.deleteTheNewestPost(true)
+  });
+
+  test("Most Liked", async ({ page }) => {
+    await buzzPage.heart.nth(1).click()
+    await buzzPage.mostLikedTab.click()
+    await expect(buzzPage.bodyTextPost.first()).toHaveText(mostLiked)
```

Comment:
How do you know that you assert post, that you liked two line above? You don't check If it is that post.

Reply:
and I don't understand what are you checking here? Why BODY text should have 'mostLiked' text? There is no sens (I'm not checking it on Localhost, maybe your version is differnet)

Reply:
I liked the post and then I changed the tab on the mostLiked post and then that post should be in first place there, do you get it now?